### ISSUE TITLE
How to test this script?

### ISSUE THREAD

**Issue by rajdude** [camerahacks/rpilocator-rss-feed#7]
Hello!
Thanks a lot for looking into my issue :-)

I just tried the same script with this URL.
````
FEED_URL = 'https://rpilocator.com/feed/?country=NL'
````
I got the same old error:
````
Traceback (most recent call last):
  File "D:\Data\Software\scripts\rpilocator-rss-pushover.py", line 60, in <module>
    control.pop()
IndexError: pop from empty list
````

For reference:
The feed only stores the last 20-25 alerts (can't remember the exact number). You are filtering the feed to US items only and there aren't any US items on the feed right now since there has been 20-25 alerts since the last US alert.

For instance, if you change the filter to NL instead of US and leave the line for control.pop() you should be able to test it.

If you find any problems, please open a new issue.

**Comment by rajdude** [comment]
I tried other regions also
CN
CH
CA

All throw the same error.

**Comment by camerahacks** [comment]
You have to wait 60 seconds between requests to my server otherwise you run the risk of getting rate limited and the feed won't return anything. Add a line to print f.status and see if you are getting a 200 status from my server.

The instruction to add control.pop() is exactly how I test this script.

**Comment by rajdude** [comment]
ok thanks a lot for the tips. 
I added a time.sleep(60) at the top of my test script, to prevent me messing up.
Then I tried adding a print (f.status) and it gives me a 200. That means it is pulling some data without problems.
List is still empty, but I guess that is because there is nothing in stock right now.

Thanks again! I think now it is a waiting game.

````
200
Traceback (most recent call last):
  File "C:\Users\rajdu\Documents\scripts\rpilocator-rss-pushover.py", line 65, in <module>
    control.pop()
IndexError: pop from empty list
````
